### DISCUSSION TITLE
Do you have a recommended migrations tool/library that you'd recommend as a partner to pydapper?

### DISCUSSION THREAD

**Discussion by preetmyob** [zschumacher/pydapper#177] [General]
In our .net code we use `dapper` and `dbup`. If I was to use `pydapper` I'm wondering if there is a good lightweight python equivalent, please?

**Comment by zschumacher** [comment]
I'm a fan of yoyo https://ollycope.com/software/yoyo/latest/

**Comment by zschumacher** [comment]
Alembic is also fine, but not quite as simple as yoyo. It supports more backends, tho. https://alembic.sqlalchemy.org/en/latest/

**Comment by preetmyob** [comment]
Thank you!
